puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/test/testKatello.groovy

```diff
@@ -16,7 +10,7 @@ pipeline {
                 ghprb_git_checkout()
 
                 dir('foreman') {
-                    git url: "https://github.com/theforeman/foreman", branch: katello_versions[ghprbTargetBranch]
+                    git url: "https://github.com/theforeman/foreman", branch: foremanVersion(ghprbTargetBranch)
```

Comment:
This doesn't append the `-stable` part. The job definition probably also needs to include `version_map.groovy`